src/components/ButtonAuth.jsx

```diff
@@ -2,19 +2,21 @@ import Image from "next/image";
 import { Button } from "react-flatifycss";
 
 import "flatifycss/dist/css/flatify-min.css";
```

Comment:
Also here this import is not needed as it's already imported in the `__app.jsx` which is going to apply for  elements beneth it.